### PR TITLE
fix(VList/VListGroup): add missing rtl margins and paddings (v2.0)

### LINKED ISSUE #8125: [Bug Report] RTL: v-list-item-action has margin-right instead of margin-left
### Environment
**Vuetify Version:** 2.0.2
**Vue Version:** 2.6.10
**Browsers:** Chrome 75.0.3770.142
**OS:** Mac OS 10.14.6

### Steps to reproduce
Set vuetify to RTL mode and add a list with list-item-action or list-item-icon slot

### Expected Behavior
expected a proper margin between the item list icon and item list content

### Actual Behavior
no margin between item list icon and item list content

### Reproduction Link
<a href="https://codepen.io/engmah/pen/oKBjwE" target="_blank">https://codepen.io/engmah/pen/oKBjwE</a>

### Other comments
I think generated css should have something like:

    .v-application--is-rtl .v-list-item__action:first-child,.v-application--is-rtl .v-list-item__icon:first-child{
      margin-right:0;
      margin-left:32px;
    }

### PR DESCRIPTION
**Pull request by melbahja**
fixes #8125

Hi,

I was testing vuetify v2 and I find paddings and margins not exists for rtl, So i decided to open this PR.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [ ] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [ ] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library) 

I'm happy to follow up and send more PRs if I find something.
Thank you :)

### CODE REVIEW

**Comment by jacekkarczmarczyk** [comment]
Please add the playground code for testing

**Comment by melbahja** [comment]
Hi, @jacekkarczmarczyk 
test it on vuetify RTL version
```vue
<template>
   <div>
      <v-card
         max-width="500"
         class="mx-auto"
       >
         <v-toolbar
           color="deep-purple accent-4"
           dark
         >
           <v-app-bar-nav-icon></v-app-bar-nav-icon>

           <v-toolbar-title>New Chat</v-toolbar-title>

           <v-spacer></v-spacer>

           <v-btn icon>
             <v-icon>search</v-icon>
           </v-btn>
         </v-toolbar>

         <v-list subheader>
           <v-subheader>Recent chat</v-subheader>

           <v-list-item
             v-for="item in items"
             :key="item.title"
             @click=""
           >
             <v-list-item-avatar>
               <v-img :src="item.avatar"></v-img>
             </v-list-item-avatar>

             <v-list-item-content>
               <v-list-item-title v-text="item.title"></v-list-item-title>
             </v-list-item-content>

             <v-list-item-icon>
               <v-icon :color="item.active ? 'deep-purple accent-4' : 'grey'">chat_bubble</v-icon>
             </v-list-item-icon>
           </v-list-item>
         </v-list>

         <v-divider></v-divider>

         <v-list subheader>
           <v-subheader>Previous chats</v-subheader>

           <v-list-item
             v-for="item in items2"
             :key="item.title"
             @click=""
           >
             <v-list-item-avatar>
               <v-img :src="item.avatar"></v-img>
             </v-list-item-avatar>

             <v-list-item-content>
               <v-list-item-title v-text="item.title"></v-list-item-title>
             </v-list-item-content>

             <v-list-item-avatar>
               <v-img :src="item.avatar"></v-img>
             </v-list-item-avatar>

           </v-list-item>
         </v-list>
       </v-card>

       <br><br>
       <v-card
         class="mx-auto"
         width="300"
       >
         <v-list>
           <v-list-item>
             <v-list-item-icon>
               <v-icon>home</v-icon>
             </v-list-item-icon>

             <v-list-item-title>Home</v-list-item-title>
           </v-list-item>

           <v-list-group
             prepend-icon="account_circle"
             value="true"
           >
             <template v-slot:activator>
               <v-list-item>
                 <v-list-item-title>Users</v-list-item-title>
               </v-list-item>
             </template>

             <v-list-group
               no-action
               sub-group
               value="true"
             >
               <template v-slot:activator>
                 <v-list-item>
                   <v-list-item-content>
                     <v-list-item-title>Admin</v-list-item-title>
                   </v-list-item-content>
                 </v-list-item>
               </template>

               <v-list-item
                 v-for="(admin, i) in admins"
                 :key="i"
                 link
               >
                 <v-list-item-title v-text="admin[0]"></v-list-item-title>
                 <v-list-item-icon>
                   <v-icon v-text="admin[1]"></v-icon>
                 </v-list-item-icon>
               </v-list-item>
             </v-list-group>

             <v-list-group
               sub-group
               no-action
             >
               <template v-slot:activator>
                 <v-list-item>
                   <v-list-item-content>
                     <v-list-item-title>Actions</v-list-item-title>
                   </v-list-item-content>
                 </v-list-item>
               </template>
               <v-list-item
                 v-for="(crud, i) in cruds"
                 :key="i"
                 @click=""
               >
                 <v-list-item-title v-text="crud[0]"></v-list-item-title>
                 <v-list-item-action>
                   <v-icon v-text="crud[1]"></v-icon>
                 </v-list-item-action>
               </v-list-item>
             </v-list-group>
           </v-list-group>
         </v-list>
       </v-card>

       <br>
       <v-card
         class="mx-auto"
         color="#26c6da"
         dark
         max-width="400"
       >
         <v-card-title>
           <v-icon
             large
             right
           >
             mdi-twitter
           </v-icon>
           <span class="title font-weight-light">Twitter</span>
         </v-card-title>

         <v-card-text class="headline font-weight-bold">
           "Turns out semicolon-less style is easier and safer in TS because most gotcha edge cases are type invalid as well."
         </v-card-text>

         <v-card-actions>
           <v-list-item class="grow">
             <v-list-item-avatar color="grey darken-3">
               <v-img
                 class="elevation-6"
                 src="https://avataaars.io/?avatarStyle=Transparent&topType=ShortHairShortCurly&accessoriesType=Prescription02&hairColor=Black&facialHairType=Blank&clotheType=Hoodie&clotheColor=White&eyeType=Default&eyebrowType=DefaultNatural&mouthType=Default&skinColor=Light"
               ></v-img>
             </v-list-item-avatar>

             <v-list-item-content>
               <v-list-item-title>Evan You</v-list-item-title>
             </v-list-item-content>

             <v-layout
               align-center
               justify-end
             >
               <v-icon class="mr-1">mdi-heart</v-icon>
               <span class="subheading mr-2">256</span>
               <span class="mr-1">·</span>
               <v-icon class="mr-1">mdi-share-variant</v-icon>
               <span class="subheading">45</span>
             </v-layout>
           </v-list-item>
         </v-card-actions>
       </v-card>
       
   </div>
</template>


<script>
export default 
{
  data: () => (
  {
    items: [
        { active: true, title: 'Jason Oner', avatar: 'https://cdn.vuetifyjs.com/images/lists/1.jpg' },
        { active: true, title: 'Ranee Carlson', avatar: 'https://cdn.vuetifyjs.com/images/lists/2.jpg' },
        { title: 'Cindy Baker', avatar: 'https://cdn.vuetifyjs.com/images/lists/3.jpg' },
        { title: 'Ali Connors', avatar: 'https://cdn.vuetifyjs.com/images/lists/4.jpg' },
      ],
      items2: [
        { title: 'Travis Howard', avatar: 'https://cdn.vuetifyjs.com/images/lists/5.jpg' },
      ],

      admins: [
        ['Management', 'people_outline'],
        ['Settings', 'settings'],
      ],
      cruds: [
        ['Create', 'add'],
        ['Read', 'insert_drive_file'],
        ['Update', 'update'],
        ['Delete', 'delete'],
      ],
    
  }),
};
</script>
```

**Comment by melbahja** [comment]
I was playing with RTL VTabs, I found issue with slide touch (```onTouchEnd``` event in ```VSlideGroup.ts```)
You can understand what I'm talking about here: 
https://codepen.io/dev0x0/full/dxoxqq
I solved the issue with this simple solution in ```VSlideGroup.ts``` line 261, I replaced scrollOffset if else with:
```
            if (this.$vuetify.rtl) {

                /* istanbul ignore else */
                if (this.scrollOffset > 0 || !this.isOverflowing) {
                    this.scrollOffset = 0;
                } else if (this.scrollOffset <= -maxScrollOffset) {
                    this.scrollOffset = -maxScrollOffset;
                }
            
            } else {

                /* istanbul ignore else */
                if (this.scrollOffset < 0 || !this.isOverflowing) {
                    this.scrollOffset = 0;
                } else if (this.scrollOffset >= maxScrollOffset) {
                    this.scrollOffset = maxScrollOffset;
                }
            }
```
@jacekkarczmarczyk @johnleider should i open a new PR or add a commit here
thank you :)

**Comment by jacekkarczmarczyk** [comment]
New PR. Ty

**Comment by jacekkarczmarczyk** [comment]
Please rebase to `master`

File: packages/vuetify/src/components/VList/VListGroup.sass [hunk 1]
```diff
@@ -50,13 +50,23 @@
   .v-list-item__icon
     &:first-child
       margin-right: 16px
+      +rtl()
```
Comment (jacekkarczmarczyk) [inline packages/vuetify/src/components/VList/VListGroup.sass]: Please use here (and in other places) ltr() function: https://github.com/vuetifyjs/vuetify/blob/b76e5e52dee99cf16c0aac3e5db5f04dc1091f5e/packages/vuetify/src/components/VTabs/VTabs.sass#L52-L57

Comment (melbahja) [reply packages/vuetify/src/components/VList/VListGroup.sass]: check it now :)

**Comment by TheInvoker** [comment]
was this change released to public already?

**Comment by melbahja** [comment]
@TheInvoker yes check v2.0.4
https://github.com/vuetifyjs/vuetify/releases/tag/v2.0.4

**Pull request merged**